doc/language_reference/language_reference.md

```diff
@@ -595,14 +605,26 @@ extern function extract_ips(addrs: string): Set<ip_addr_t>
 ```
 
 The sixth form groups records computed so far by a subset of fields,
-computes an aggreagate for each group using specified aggregate function
-(*requirements on aggregate function signature to be specified*), and
-binds the result to a new variable:
+computes an aggreagate for each group using specified aggregate function,
```

Comment:
I wonder whether we should not call it GroupByAndAggregate. It is more in line with its semantics, although it's a bit long.